.github/workflows/docker-publish.yml

```diff
@@ -1,39 +0,0 @@
-name: Docker Publishing
```

Comment:
How come we have removed this workflow?

Reply:
Because it is now easy courtesy of `make` to build the images locally, which is in some ways more desirable. One less thing, and fewer resources used every damn time we push a commit.